birdhouse/env.local.example

```diff
@@ -104,6 +103,11 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
+# The parent folder where all the user notebooks will be stored.
+# For example, a user named "bob" will have his data in $JUPYTERHUB_USER_DATA_DIR/bob
+# and this folder will be mounted when he logs into JupyterHub.
+export JUPYTERHUB_USER_DATA_DIR=/data/jupyterhub_user_data
```

Comment:
This line should be commented out and a default value added to `./pavics-compose.sh`.  The whole point is to make the existence of optional var optional in `env.local`.

Reply:
Done